Cargo.toml

```diff
@@ -16,3 +16,8 @@ rrg-proto = { path = "proto/" }
 simplelog = { version = "0.7.6" }
 structopt = { version = "0.3.12" }
 sys-info = { version = "0.5.1" }
+tempfile = { version = "3.1.0" }
+users = { version = "0.10.0" }
```

Comment:
Yes, they are. Since `users` is unix-specific crate I think I should place it into already defined section of dev-dependencies with `target_os = "linux"` conditional compilation restriction.